tekton/koparse/koparse.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python2.7
```

Comment:
ohh fancy

Reply:
Why not `python3` though :trollface:

Reply:
lol you raise a good point @vdemeester XD